cloudbuild.yaml

```diff
@@ -1,119 +1,87 @@
 steps:
 
-# Initializations
-
-- id: Initialize git
-  name: gcr.io/cloud-builders/git
-  entrypoint: /bin/bash
-  args:
-  - -exc
-  - |
-    # Cloud Build x GitHub integration uses source archives to fetch
-    # the source, rather than Git source fetching, and as a consequence
-    # does not include the .git/ directory. As a workaround, we clone
-    # the repository and reset it to this build's commit sha.
-    git clone 'https://github.com/GoogleCloudPlatform/marketplace-k8s-app-example.git' tmp
-    mv tmp/.git .git
-    rm -rf tmp
-    git reset "$COMMIT_SHA"
-    git submodule sync --recursive
-    git submodule update --init --recursive
-
-- id: Get Kubernetes Credentials
-  name: gcr.io/cloud-builders/gcloud
-  args:
-  - container
-  - clusters
-  - get-credentials
-  - "limani-integ"
-  - --region
-  - "us-central1"
-  - --project
-  - "$PROJECT_ID"
-
-- id: Pull Dev Image
+- id: &PullDevImage Pull Dev Image
   name: gcr.io/cloud-builders/docker
   waitFor:
-  - Initialize git
-  dir: vendor/marketplace-tools
+  - '-'
   entrypoint: bash
   args:
   - -exc
   - |
-    TAG="$$(./scripts/derive_tag.sh)"
+    TAG="$$(cat MARKETPLACE_TOOLS_TAG)"
     docker pull "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG"
-    docker tag "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/dev:local"
+    docker tag "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/dev:build-$BUILD_ID"
```

Comment:
since we never pushed these images, would recommend using a static tag here like `local` instead